Post release doc update (#817)
oamg/leapp: docs/source/el7toel8/envars.md

## LEAPP_NO_RHSM

> Do not use Red Hat Subscription Management for the upgrade, if set 1.

Reviewer: ```suggestion
Do not use Red Hat Subscription Management for the upgrade, if set to 1.
```
I would do this in the other options too

in malfunction network configuration (e.g. in case the bonding is configured
on the system). It's expected that NICs have to be handled manually if needed.

## LEAPP_DATABASE_FORCE_SYNC_ON

> If set to 1, leapp will explicitly enable synchronization on the SQLite database.

Reviewer 2: ```suggestion
If set to 1, Leapp will explicitly enable synchronization on the SQLite database.
```

Enabling the synchronization has negative impact on the performance
(sometimes very negative) however it is more reliable in case of extreme
situations (e.g. lost power).
Note the synchronization is nowadays switched off by default only during the phases

> executed before the reboot of the system to the upgrade environment which we consider

Reviewer 2: ```suggestion
executed before the reboot of the system to the upgrade environment, which we consider
```

> safe, so we do not expect that someone would want to use this option now.

Reviewer 2: ```suggestion
safe. As a result, we do not expect that someone would want to use this option now.
```

> However, we are testing and discussing the possibility to switch off

Reviewer 2: Are our future plans necessary for customers to know about?

Author: @mportman12  I am not looking at it as an information shared with customers, but an information shared with the community (which could include customers if they read it). From that point it should be ok to mention it. It's probably not necessary, but I felt it as a good thing to add - it's possible that same good decision could be to drop it, as it has possibly low value :D  I am still thinking whether actually to document the envar itself or drop it completely.

Reviewer 2: @pirat89 Ok, in that case recommend the following rewrite. The current text is made up of very long sentences, so it's hard to follow the train of thought:
"Note the synchronization is currently disabled by default only during the phases
executed before the reboot of the system to the upgrade environment. Because this is a safe phase to disable synchronization, Red Hat does not anticipate many users will want to re-enable synchronization. 
However, Red Hat is currently testing and discussing the possibility of disabling the synchronization for all phases of the upgrade. If synchronization becomes disabled by default for the entire upgrade, users can use this environment variable for troubleshooting purposes."

Author: @mportman12  I like this, just I will exchange here Red Hat by "we" in the upstream documentation. But it sounds better now. I will update it during the start of the week. thanks!!

Author: dropping the part about the future plans.

> If set to 1, leapp will explicitely synchronization on the SQLite database.
> Enabling the synchronisation has negative impact on the performance
> (sometimes very negative) however the data is more reliable in case of extreme
> situations (e.g. lost power).
> Note the synchronisation is nowadays switched off by default only during the phases
> executed before the reboot of the system to the upgrade environment which we consider
> safe, so we do not expect that anyone will want to use this option now.
> However, we are testing and discussing the possibility to switch off
> the syncrhonisation during the whole upgrade procedure in future

Reviewer: ```suggestion
If set to 1, leapp will explicitly enable synchronization on the SQLite database.
Enabling the synchronization has negative impact on the performance
(sometimes very negative) however the data is more reliable in case of extreme
situations (e.g. lost power).
Note the synchronization is nowadays switched off by default only during the phases
executed before the reboot of the system to the upgrade environment which we consider
safe, so we do not expect that anyone will want to use this option now.
However, we are testing and discussing the possibility to switch off
the syncrhonization during the whole upgrade procedure in future
```

Reviewer: Adding fixup commit

-  then, in such a case it's possible someone would want to use it, but mostly
only for the investigation purposes in case of critical failures, to see
what last operations have been done before the failure.

## LEAPP_NO_INSIGHTS_REGISTER

> Do not register the system into Red Hat Insights automatically, if set to 1.

Reviewer 2: ```suggestion
If set to 1, Leapp does not register the system into Red Hat Insights automatically.
```

> It's equivalent to the `--no-rhsm-facts` leapp option.

Reviewer: ```suggestion
It's equivalent to the `--no-insights-register` leapp option.
```

## LEAPP_NO_RHSM_FACTS

> Do not store migration information using Red Hat Subscription Manager, if set to 1.

Reviewer 2: ```suggestion
If set to 1, Leapp does not store migration information using Red Hat Subscription Manager.
```

It's equivalent to the `--no-rhsm-facts` leapp option.

## LEAPP_NOGPGCHECK

> Disable RPM GPG checks (same as yum/dnf --nogpgckeck option).

Reviewer 2: ```suggestion
Set to 1 to disable RPM GPG checks (same as yum/dnf --nogpgckeck option).
```

database. The positive effect is significant speed up of the leapp execution,
however it comes at the cost of risking a corrupted database, so it is
currently used for testing / development purposes, only.

## LEAPP_DEVEL_INITRAM_NETWORK
You can specify one of the following values: 'network-manager', 'scripts'.
The 'scripts' value is used for a legacy dracut module when the network is not
handled by NetworkManager.

> Using the option allows experimental upgrades, bringin up the networking inside

Reviewer: ```suggestion
Using the option allows experimental upgrades, bringing up the networking inside
```
